napari/layers/shapes/shapes.py

```diff
@@ -1622,6 +1688,23 @@ def _validate_properties(
 
         return properties
 
+    @property
+    def text(self) -> TextManager:
+        """TextManager: The TextManager object containing the text properties"""
+        return self._text
+
+    @text.setter
+    def text(self, text):
+        self._text._set_text(
+            text, n_text=len(self.data), properties=self.properties
+        )
+
+    def refresh_text(self):
+        """Refresh the text values.
+        This is generally used if the properties were updated without changing the data
```

Comment:
```suggestion

        This is generally used if the properties were updated without changing the data
```

This doesn't feel like it should be a public method, should it?